elide-core/src/main/java/com/yahoo/elide/resources/JsonApiEndpoint.java

```diff
@@ -112,13 +113,29 @@ public Response patch(
      */
     @DELETE
     @Path("{path:.*}")
+    @Consumes("application/vnd.api+json")
     public Response delete(
         @PathParam("path") String path,
         @Context SecurityContext securityContext,
         String jsonApiDocument) {
         return build(elide.delete(path, jsonApiDocument, getUser.apply(securityContext)));
     }
 
+    /**
+     * Delete resource handler (expects no body and no content-type header).
+     *
+     * @param path request path
+     * @param securityContext security context
+     * @return response
+     */
+    @DELETE
+    @Path("{path:.*}")
+    public Response deleteWithoutBody(
```

Comment:
I'm not sure `@Consumes("")` is allowed by JAX-RS specification. Even if something like this will work in jersey it can be not supported by resteasy etc. 

Afaik the only way to tell jax-rs that you want no request body - is to omit consumes annotation and body method parameter.